components/Text/PositionType.tsx

```diff
@@ -1,37 +1,57 @@
 import styled, { css } from 'styled-components';
 
+import { PositionSide } from 'queries/futures/types';
+
 type PositionProps = {
-	side: 'long' | 'short';
+	side: PositionSide;
+	mobile?: boolean;
 };
 
-const PositionType: React.FC<PositionProps> = ({ side = 'long' }) => {
-	return <StyledText $side={side}>{side}</StyledText>;
+const PositionType: React.FC<PositionProps> = ({ side = PositionSide.LONG, mobile = false }) => {
+	return mobile ? (
+		<MobileStyledText side={side}>{side}</MobileStyledText>
+	) : (
+		<StyledText side={side}>{side}</StyledText>
+	);
 };
 
-const StyledText = styled.p<{ $side: PositionProps['side'] }>`
+const StyledText = styled.p<{ side: PositionSide }>`
 	text-transform: uppercase;
 	padding: 5px 9px;
 	border-radius: 6px;
 
 	${(props) =>
-		props.$side === 'long' &&
-		css`
-			color: ${(props) => props.theme.colors.selectedTheme.green};
-			background: rgba(127, 212, 130, 0.1);
-			font-family: ${(props) => props.theme.fonts.monoBold};
-			font-variant: all-small-caps;
-			letter-spacing: 1.4px;
-		`};
+		props.side === PositionSide.LONG
+			? css`
+					color: ${(props) => props.theme.colors.selectedTheme.green};
+					background: rgba(127, 212, 130, 0.1);
+					font-family: ${(props) => props.theme.fonts.monoBold};
+					font-variant: all-small-caps;
+					letter-spacing: 1.4px;
+			  `
+			: css`
+					color: ${(props) => props.theme.colors.selectedTheme.red};
+					background: rgba(239, 104, 104, 0.1);
+					font-family: ${(props) => props.theme.fonts.monoBold};
+					font-variant: all-small-caps;
+					letter-spacing: -0.2px;
+			  `};
+	};
+`;
+
+const MobileStyledText = styled.p<{ side: PositionSide }>`
+	margin: 0;
+	text-transform: uppercase;
+	font-family: ${(props) => props.theme.fonts.bold};
 
 	${(props) =>
```

Comment:
Also a bit boated here, could reduce to:
```
color: ${(props) => props.side === PositionSide.LONG ? props.theme.colors.common.primaryGreen : props.theme.colors.common.primaryRed}
```

Reply:
fixed this and one other one I hadn't noticed where we repeated the styles for both sides.